go.work

```diff
@@ -1,3 +1,4 @@
 go 1.20
```

Comment:
as go.work files aren't intended to be shared I think we should pull this out of this PR